modules/base/rendering/renderableplane.cpp

```diff
@@ -262,6 +316,33 @@ void RenderablePlane::render(const RenderData& data, RendererTasks&) {
     cameraOrientedRotation[1] = glm::dvec4(newUp, 0.0);
     cameraOrientedRotation[2] = glm::dvec4(normal, 0.0);
 
+    if (_scaleByDistance) {
+        const glm::dvec3 cameraPosition = data.camera.positionVec3();
+        const glm::dvec3 modelPosition = data.modelTransform.translation;
+
+        const float fovDegrees = global::windowDelegate->getHorizFieldOfView();
```

Comment:
This fails in Fisheye mode, where the horizontal field of view returned is not valid.  I'd look at the RenderablePointCloud class to see how it does the scaling and if you can do it without needing the FOV here.  @WeirdRubberDuck  probably has more insight on this

Reply:
Here's the code that does it for the points, using the position of the camera.  The angle that you have here is computed in world space from the position of the camera, the object, and a vector describin the size of the object

https://github.com/OpenSpace/OpenSpace/blob/17279d2b4ad8b1672087c04e0742fbea2e460107/modules/base/shaders/pointcloud/pointcloud_gs.glsl#L135

Let me know if you want to discuss it @lundkvistarn !

Reply:
Also, I think that it might maybe be worth to move this computation to a geometry shader. It's currently not that smooth when interacting. See this video

https://github.com/user-attachments/assets/89d6363c-3345-45be-980e-eb2688a3224b